ambry-store/src/test/java/com.github.ambry.store/IndexValueTest.java

```diff
@@ -81,6 +81,18 @@ public void putValueTest() throws InterruptedException {
     // max value
     expirationTimeAtMs = TimeUnit.SECONDS.toMillis(Integer.MAX_VALUE);
     expirationTimes.put(expirationTimeAtMs, Utils.getTimeInMsToTheNearestSec(expirationTimeAtMs));
+    // expiry > Integer.MAX_VALUE, expected to be -1
+    expirationTimeAtMs = TimeUnit.SECONDS.toMillis((long) Integer.MAX_VALUE + 1);
+    expirationTimes.put(expirationTimeAtMs, Utils.Infinite_Time);
+    // expiry > Integer.MAX_VALUE, expected to be -1
+    expirationTimeAtMs = TimeUnit.SECONDS.toMillis(2 * (long) Integer.MAX_VALUE);
+    expirationTimes.put(expirationTimeAtMs, Utils.Infinite_Time);
+    // expiry < 0
+    expirationTimeAtMs = -1 * TimeUnit.DAYS.toMillis(1);
+    expirationTimes.put(expirationTimeAtMs, Utils.getTimeInMsToTheNearestSec(expirationTimeAtMs));
+    // expiry < 0
+    expirationTimeAtMs = -1 * (long) Integer.MAX_VALUE;
```

Comment:
Any reason to test this value instead of `Integer.MIN_VALUE`? (-(2^31 - 1) vs -(2^31))